Modules/Core/Support/DiacriticsHelper.php

```
@@ -0,0 +1,310 @@
+ * Static helper class converted from procedural functions.
+ */
+class DiacriticsHelper
+{
+    /**
+     * @param string $str
+     */
+    public static function diacritics_seems_utf8($str): bool
+    {
+        $length = mb_strlen($str);
+        for ($i = 0; $i < $length; $i++) {
+            $c = ord($str[$i]);
```

PHPMD (2.15.0)

Avoid variables with short names like $c. Configured minimum length is 3. (undefined)

(ShortVariable)